suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml

```diff
@@ -0,0 +1,3 @@
+tasks:
```

Comment:
This should be appended to other tasks in tasks list for better readablity just like we have it for other smoke suite